code_schemes/ws_correct_dataset.json

```diff
@@ -0,0 +1,34 @@
+{
+    "SchemeID": "Scheme-fffffb5c",
+    "Name": "WS - Correct Dataset",
+    "Version": "0.0.0.1",
+    "Codes": [
+        {
```

Comment:
This should contain all the ws codes in ke vax plus the new datasets for this project.